frontend/src/pages/Admin/Locations.tsx

```diff
@@ -15,6 +15,11 @@ const Locations = () => {
       const locationsData = await fetch('/api/locations', withDefaults())
         .then((res) => res.json())
         .then((data) => data.data);
+      locationsData.sort((a: Location, b: Location) => {
+        if (a.name < b.name) { return -1; }
+        if (a.name > b.name) { return 1; }
+        return 0;
```

Comment:
Super minor, but could this be combined into something like `a.name.localeCompare(b.name)` ? (not 100% sure if that's the function)

Reply:
apparently localeCompare is much slower, but nice catch! i didnt even know that existed until now lol